torch_np/_detail/_unary_ufuncs.py

```diff
@@ -0,0 +1,55 @@
+"""Export torch work functions for unary ufuncs, rename/tweak to match numpy.
+This listing is further exported to public symbols in the `torch_np/_unary_ufuncs.py` module.
+"""
+
+import torch
+
+# renames
```

Comment:
Why the different imports here, rather than just do a `from torch import ( blablalba )`?

Reply:
This is `isort` from `pre-commit run`.

Reply:
Terribly weird. @honno any idea what's going on here?